src/main/java/org/shanerx/tradeshop/utils/objects/ObjectHolder.java

```diff
@@ -115,6 +121,21 @@ public Map<String, Object> asMap() {
         return null;
     }
 
+    public boolean isList() {
+        return !isNull() && obj instanceof List;
```

Comment:
not sure this works... you need to be careful with generics
`class ObjectHolder<T>` means you need to check `obj instanceof List<? extends T>`
this is not the same, `List<String>` is not compatible or a subclass of `List<Object>`

Reply:
you right, thats a goof.

Well, it shouldn't be ? extends T because the T would be a list(and most likely a String List)

Reply:
Yes you're right, I meant T is whatever  obj is a list of... maybe instanceof List<?> should be fine then?